Library/Homebrew/brew.sh

```diff
@@ -174,9 +174,9 @@ fi
 
 if [[ -n "$HOMEBREW_MACOS" || -n "$HOMEBREW_FORCE_HOMEBREW_ON_LINUX" ]]
 then
-  HOMEBREW_BOTTLE_DEFAULT_DOMAIN="https://homebrew.bintray.com"
+  export HOMEBREW_BOTTLE_DEFAULT_DOMAIN="https://homebrew.bintray.com"
```

Comment:
Can you export on line 221 instead? Thanks!

Reply:
Done.